src/app/layers/layersActions.js

```diff
@@ -177,7 +179,11 @@ export function initLayers(workspaceLayers, libraryLayers) {
     // get header promises
     const headersPromises = []
     workspaceLayers
-      .filter((l) => LAYER_TYPES_WITH_HEADER.includes(l.type) && l.added === true)
+      .filter(
+        (l) =>
+          (l.id === 'encounters_ais' || LAYER_TYPES_WITH_HEADER.includes(l.type)) &&
```

Comment:
Why not move this to a constant? as it is used also here https://github.com/GlobalFishingWatch/map-client/pull/1029/files#diff-409f4d26ff67e6335b043fc86967e616R10 and several places it could be an issue in the future